.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+## Title
+<!-- Enter your ticket number and text, and make a link to it -->
+
+## Describe your changes: 
+<!-- Describe what changes you did, and include screenshots if necessary -->
+
+## Type of changes:
+<!-- What type of changes did you make? -->
+
+## Checklist before requesting a review
+
+* [ ] I created a new branch before starting with this ticket?
+* [ ] I commented the issue after finish in the Frontend Team Board?
+* [ ] I moved the card to Quality Assurance after finish?
+
+## Did you run into any issues?
+
+* [ ] No
+* [ ] Yes I will describe them below:
```

Comment:
When using checkboxes in markdown, GitHub interprets each checkbox as a task and uses it to track progress as can be seen here in your PR - it gets marked as 5 of 6 tasks completed.

![image](https://github.com/NoroffFEU/agency.noroff.dev/assets/93515165/6d355122-81d5-479f-97dc-3eab3b58565b)

I would suggest not using checkboxes for this section, and instead using descriptive text
Something like this perhaps:

![image](https://github.com/NoroffFEU/agency.noroff.dev/assets/93515165/2629eeb7-1a07-4910-b62b-42fa18e92554)

Reply:
Oh, I wasn't aware that all boxes need to be checked off to achieve 100% completion. I will fix this.